File: SDKs/amazonaws.com/route53-recovery-control-config/2020-11-02/typescript/src/sdk/models/shared/listsafetyrulesresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { Rule } from "./rule";



export class ListSafetyRulesResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=NextToken" })
  nextToken?: string;

  @SpeakeasyMetadata({ data: "json, name=SafetyRules", elemType: Rule })
  safetyRules?: Rule[];
}

File: SDKs/amazonaws.com/route53-recovery-control-config/2020-11-02/python/src/sdk/models/shared/listsafetyrulesresponse.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from sdk import utils
from . import *


@dataclass_json
@dataclass
class ListSafetyRulesResponse:
    next_token: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('NextToken') }})
    safety_rules: Optional[List[Rule]] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('SafetyRules') }})
    
